docs/daemon/network.mdx

```diff
@@ -0,0 +1,169 @@
+---
+sidebar_position: 4
+sidebar_label: "Testnet"
+id: daemon-network-setup
+title: Network Setup
+description: Deploying a Kwil Network with multiple nodes
+slug: /daemon/network/setup
+---
+
+This guide provides step-by-step instructions for configuring and deploying a Kwil network with an initial set of M validator and N non-validator nodes. 
+
+## Network Configuration
+The `setup testnet` command can be used to generate the configuration required for multiple nodes, all with the same genesis configuration, and pre-configured to connect to each other. Refer to the [testnet deployment guide](./../admin/setup#creating-a-new-test-network-of-several-nodes) for more information on the command usage. 
```

Comment:
I suppose we should consider changing the command to be `setup network` since there's really no testnet.

Reply:
agreed